gradle/libs.versions.toml

```diff
@@ -106,6 +106,7 @@ okhttp-mockwebserver = { module = "com.squareup.okhttp3:mockwebserver", version.
 playServices-ads-identifier = { module = "com.google.android.gms:play-services-ads-identifier", version.ref = "adsIdentifier" }
 
 robolectric = { module = "org.robolectric:robolectric", version.ref = "roboelectric" }
+hamcrest-core = { module = "org.hamcrest:hamcrest-core", version = "1.3" }
```

Comment:
can you add the version to the top of the file and reference to it here?

Reply:
Definitely! Wasn't sure if there's a convention, as I saw both `version.ref` and `version` being used.

Reply:
I think we try to do `version.ref`, but Android Studio doesn't so some non-refs might have sneaked in the file lol

Reply:
Ha okay good to know 😄